README.md

```diff
@@ -40,17 +41,19 @@ Follow these steps to successfully add a new workflow:
 See [Continuous Integration with make](https://github.com/parodos-dev/serverless-workflows/blob/main/make.md) for implementation details of the CI pipeline.
 
 ### Builder image
+
 There are two builder images under ./pipeline folder:
+
 - workflow-builder-dev.Dockerfile - references nightly build image from `docker.io/apache/incubator-kie-sonataflow-builder:main` that doesn't required any authorization
 - workflow-builder.Dockerfile - references OpenShift Serverless Logic builder image from registry.redhat.io which requires authorization.
   - To use this dockerfile locally, you must be logged to registry.redhat.io. To get access to that registry, follow:
     1. Get tokens [here](https://access.redhat.com/terms-based-registry/accounts). Once logged in to podman, you should be able to pull the image.
     2. Verify pulling the image [here](https://catalog.redhat.com/software/containers/openshift-serverless-1-tech-preview/logic-swf-builder-rhel8/6483079349c48023fc262858?architecture=amd64&image=65e1a56104e00058ecdd52eb&container-tabs=gti)
 
-
 Note on CI:
 For every PR merged in the workflow directory, a GitHub Action runs an image build to generate manifests, and a new PR is automatically generated in the [serverless-workflows-config repository][3]. The credentials used by the build process are defined as organization level secret, and the content is from a token on the helm repo with an expiry period of 60 days. Currently only the repo owner (rgolangh) can recreate the token. This should be revised.
 
 [1]: https://github.com/serverlessworkflow/specification/blob/main/specification.md
 [2]: https://github.com/apache/incubator-kie-kogito-serverless-operator/
 [3]: https://github.com/parodos-dev/serverless-workflows-config
+[4]: ./best-practices.md
```

Comment:
@mareklibra pls replace this with a full URL since relative URLs will fail to render in the website.

Reply:
done